packages/block-library/src/navigation/edit/index.js

```diff
@@ -707,9 +704,17 @@ function Navigation( {
 									handleUpdateMenu( menuId );
 									setShouldFocusNavigationSelector( true );
 								} }
-								onSelectClassicMenu={ ( classicMenu ) => {
-									convert( classicMenu.id, classicMenu.name );
-									setShouldFocusNavigationSelector( true );
+								onSelectClassicMenu={ async ( classicMenu ) => {
```

Comment:
Would it be worth extracting this to a shared method since it is the same as the above event handler?

Reply:
Thanks for reviewing. The final code is different in https://github.com/WordPress/gutenberg/pull/42956, where there will be an extra line in one of the callbacks.

Extracting to another function is still possible, but I'm not sure it's worth it for only a few lines. I personally prefer the improved readability gained by not having to scroll to the definition.